widget/embedded/src/pages/SettingsPage.tsx

```diff
@@ -47,7 +47,11 @@ export function SettingsPage() {
             )}
             action={
               <ResetAction>
-                <Button type="secondary" size="small" onClick={onClick}>
+                <Button
+                  id="widget-setting-reset-btn"
+                  type="secondary"
+                  size="small"
+                  onClick={onClick}>
```

Comment:
i think its better to use 'exit-campaign-mode' name for the action

Reply:
fixed